src/bp/core/routers/auth.ts

```diff
@@ -107,12 +108,12 @@ export class AuthRouter extends CustomRouter {
               .min(0)
               .max(35)
               .trim()
-              .required(),
+              .allow(''),
```

Comment:
Why allow empty string instead of marking the first/last names as required?

Reply:
By default you have no name, but once you set one, you can't remove it (or just set your first name).
Just leaving the possibility to remove it once it has been changed. Before we required a user to fill in his profile, but now it's no longer the case